static/app/views/releases/detail/overview/sidebar/projectReleaseDetails.tsx

```diff
@@ -34,6 +50,62 @@ function ProjectReleaseDetails({release, releaseMeta, projectSlug}: Props) {
             keyName={t('Created')}
             value={<DateTime date={dateCreated} seconds={false} />}
           />
+          <KeyValueTableRow
+            keyName={
+              <Flex gap={space(0.75)} align="center">
+                {t('Finalized')}
+                <Tooltip
+                  skipWrapper
+                  title={tct(
+                    'By default a release is created "unreleased". Finalizing a release means that we populate a second timestamp on the release record, which is prioritized over [code:date_created] when sorting releases. [docs:Read More].',
+                    {
+                      br: <br />,
+                      code: <code />,
+                      docs: (
+                        <ExternalLink href="https://docs.sentry.io/cli/releases/#finalizing-releases" />
+                      ),
+                    }
+                  )}
+                >
+                  <IconInfo />
+                </Tooltip>
+              </Flex>
+            }
+            value={
+              dateReleased ? (
+                <DateTime date={dateReleased} seconds={false} />
+              ) : (
+                <Tooltip
+                  title={t(
+                    'Set release date to %s',
+                    moment
+                      .tz(
+                        release.firstEvent ?? release.dateCreated,
+                        options?.timezone ?? ''
+                      )
+                      .format(
+                        options?.clock24Hours
+                          ? 'MMMM D, YYYY HH:mm z'
+                          : 'MMMM D, YYYY h:mm A z'
+                      )
+                  )}
+                >
+                  <Button
+                    size="xs"
+                    onClick={() => {
+                      finalizeRelease.mutate([release], {
+                        onSettled() {
+                          window.location.reload();
```

Comment:
😬

Reply:
Old pages are old. I didn't invest much time here because we do still plan to remove these pages, in the interim we'll see if people even use this 'new' feature.